CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All changes that impact users of this module are documented in this file, in the [Common Changelog](https://common-changelog.org) format with some additional specifications defined in the CONTRIBUTING file. This codebase adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased [no-release]
```

Comment:
When you consider that a release is not needed you have to follow [these guidelines](https://github.com/OpenTermsArchive/engine/blob/main/CONTRIBUTING.md#changelog)